server/src/test/java/org/elasticsearch/indices/IndexingMemoryControllerTests.java

```diff
@@ -432,20 +432,20 @@ protected long getShardWritingBytes(IndexShard shard) {
             }
         };
         int iterations = randomIntBetween(10, 100);
+        ThreadPoolStats.Stats beforeStats = getRefreshThreadPoolStats();
         for (int i = 0; i < iterations; i++) {
             controller.forceCheck();
         }
         assertBusy(() -> {
             ThreadPoolStats.Stats stats = getRefreshThreadPoolStats();
-            assertThat(stats.getQueue(), equalTo(0));
-            assertThat(stats.getActive(), equalTo(1));
+            assertThat(stats.getCompleted(), equalTo(beforeStats.getCompleted() + iterations - 1));
         });
         refreshLatch.get().countDown(); // allow refresh
         assertBusy(() -> {
             ThreadPoolStats.Stats stats = getRefreshThreadPoolStats();
-            assertThat(stats.getQueue(), equalTo(0));
-            assertThat(stats.getActive(), equalTo(0));
+            assertThat(stats.getCompleted(), equalTo(beforeStats.getCompleted() + iterations));
         });
+        System.out.println(shard.refreshStats().getTotal());
```

Comment:
I guess this is a leftover.

Reply:
Yes, thanks, fixed in 1b8867eba3c804db2852bed662c16248c5b0d79c